src/utils/convert.js

```diff
@@ -150,20 +160,43 @@ ${fields.join(',\n')}
     ]`;
 };
 
+// Get some quick metadata on auth scripting methods
+const getAuthMetaData = (definition) => {
+  const js = definition.js ? stripComments(definition.js) : '';
+
+  const hasPreOAuthTokenScripting = js.indexOf('pre_oauthv2_token') > 0;
+  const hasPostOAuthTokenScripting = js.indexOf('post_oauthv2_token') > 0;
+  const hasPreOAuthRefreshScripting = js.indexOf('pre_oauthv2_refresh') > 0;
+  const hasGetConnectionLabelScripting = js.indexOf('get_connection_label') > 0;
+
+  return {
+    hasPreOAuthTokenScripting,
+    hasPostOAuthTokenScripting,
+    hasPreOAuthRefreshScripting,
+    hasGetConnectionLabelScripting,
+  };
+};
+
+const getTestTriggerKey = (definition) => {
+  return _.get(definition, ['general', 'test_trigger_key']);
```

Comment:
FYI: This can be nothing (`null` is what would come from the API, I believe) if the app has no auth. I don't think you need to change anything (we simply shouldn't render/create auth if the app had none)